protocol/x/clob/memclob/memclob.go

```diff
@@ -31,13 +32,8 @@ import (
 var _ types.MemClob = &MemClobPriceTimePriority{}
 
 type MemClobPriceTimePriority struct {
-	// ---- Fields for open orders ----
-	// Struct for storing all open orders (including their expiries).
-	openOrders *memclobOpenOrders
-
-	// ---- Fields for canceled orders ----
-	// Struct for storing order cancelations (including their expiries).
-	cancels *memclobCancels
+	// Holds every `Orderbook` by ID of the CLOB.
+	orderbooks map[types.ClobPairId]*Orderbook
```

Comment:
I actually think it's better to have `map[types.ClobPairId]MemClobPriceTimePriority`, and each memclob will just maintain one orderbook instead of a map of orderbooks 

mostly because I want to move `operationsToPropose` to the keeper so that we don't have to round trip in certain cases. e.g. deleveraging doesn't modify the memclob at all, but we still need to call functions like keeper -> memclob -> keeper, in order to add the deleveraging operation to the operations queue.

Reply:
not a big deal since you already moved stuff to `Orderbook` and it's essentially the same idea. also the design of clob is subject to change in the short to medium term so let's just go ahead with your approach